whitepapers/Tables.md

```diff
@@ -1,5 +1,106 @@
 # SQL Tables
 
+## Template
 
+### {ModelName}
+{Description}
 
+#### Columns
+* `{column_name}` -- {description} ({datatype}, [unique,] [indexed,] [nullable,] [default={value},] [choices={choices],])
```

Comment:
Oh right, tables are a thing! That's way better. I like blank for false and an icon for true (maybe ✓), so the eye is drawn to the true values.